### ISSUE TITLE
人噢波特

### ISSUE THREAD

**Issue by Chortine** [facebookresearch/home-robot#332]

**Comment by yvsriram** [comment]
Can you please add a visualization and share more details (scene id/episode id)?

**Comment by yvsriram** [comment]
The `train` episodes do not come with valid episode inits. You can append `habitat.task.episode_init=False` to your command to randomly sample a valid init.

**Comment by cpaxton** [comment]
@yvsriram should we open a PR to make this a default?

**Comment by cpaxton** [comment]
@Chortine is the issue resolved?

**Comment by Chortine** [comment]
> @Chortine is the issue resolved?

Yes, it now can run. Thanks !

**Comment by Chortine** [comment]
> > @Chortine is the issue resolved?
> 
> Yes, it now can run. Thanks !

The command I run is:
`python projects/habitat_ovmm/eval_baselines_agent.py habitat.dataset.episode_ids="[128]" habitat.dataset.split="train" habitat.task.episode_init=False`

However, it keeps producing this warning:

![image](https://github.com/facebookresearch/home-robot/assets/107395103/67b9f41c-3e52-4e3a-9e61-c62fc8d5a4ff)

**Comment by yvsriram** [comment]
Yes, I am seeing this on my end as well when running eval with **train** episodes. The spawning seems fine but my guess is that start position from episode is being used as an endpoint for shortest path computation somewhere (mostly outside of OVMM code).

I will check:
1. where this is.
2. if it will affect OVMM evaluations

Meanwhile, feel free to use `HABITAT_SIM_LOG=quiet` to disable warnings

**Comment by yvsriram** [comment]
Pushed a fix for this and also removed `episode_init` option (you needn't set/unset it anymore). Let me know if you are still facing issues running evaluations on `train`.

**Comment by cpaxton** [comment]
@Chortine is your issue stiill a problem?

**Comment by cpaxton** [comment]
Closing as we pushed a fix and there's no further updates
